### DISCUSSION TITLE
bug: <Walletconnect v2 without themeConfig>

### DISCUSSION THREAD

**Discussion by StKostyantin** [wagmi-dev/references#171] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Currently, the WalletConnectConnector accepts the following parameters 

```
constructor(config: {
        chains?: Chain[];
        options: WalletConnectOptions;
  });
```

According to these [documents](https://docs.walletconnect.com/2.0/web3modal/configuration#general-options)

### Expected Behavior

But the documentation also includes theme customization([themeConfig](https://docs.walletconnect.com/2.0/web3modal/configuration#theming-options)). 

### Steps To Reproduce

It would be nice to add this config for more detailed customization for WalletConnectConnector. At least the `themeZIndex` parameter

**Comment by jxom** [comment]
@0xAsimetriq – does `@walletconnect/ethereum-provider` have support for this?

**Comment by xzilja** [comment]
Some options are not yet exposed through `ethereum-provider`, but we will add them in soon 👍

**Reply by yiming0** [reply]
😊 this options are available now.

👉https://github.com/WalletConnect/walletconnect-monorepo/blob/827863d2ee7959fb5ba005a69ef94e1d7e45d941/providers/ethereum-provider/src/EthereumProvider.ts#L202

**Comment by jxom** [comment]
Converting to discussions as this is a feature request.
